README.md

```diff
@@ -33,6 +33,21 @@ config :bugsnag, use_logger: true
 
 # Override the default bugsnag endpoint url
 config :bugsnag, endpoint_url: "https://notify.bugsnag.com"
+
+# Tell bugsnag to mark stack frames containing "myproj" as in-project
+config :bugsnag, in_project: "myproj"
+
+# Tell bugsnag to mark stack frames starting with "lib/myproj" as in-project
```

Comment:
Is "starting with" accurate here? I'd think the regex would need to be `^lib/myproj` for that.